<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1063</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1063</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QComboBox" name="comboBox_leaveStyle">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>86</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>感染新冠</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>发热咳嗽发力咽痛腹泻等疑似新冠症状</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>病假（非新冠或非疑似新冠生病）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实习见习</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>求职择业</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>求职考试</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>专业技能考试</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>因其他个人原因请假</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>80</y>
       <width>371</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_leaveReason">
        <property name="text">
         <string>请假原因</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_leaveReason">
        <property name="placeholderText">
         <string>请输入请假原因</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ptn_submit">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>300</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>提交请假</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>182</y>
       <width>311</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_dateEnd">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_end">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>选择日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_end"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>134</y>
       <width>311</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_dateBegin">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_begin">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>选择日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_begin"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_leaveStyle">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>90</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请假类型</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_isLeave">
        <property name="text">
         <string>需要离校</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ImageSwitch" name="imageSwitch" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>69</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">QPushButton#ptn_last
{
	image: url(:/images/left_arrow.png);
}
QPushButton#ptn_next
{
	image: url(:/images/right_arrow.png);
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>505</width>
       <height>47</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="label_total">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="ptn_home">
          <property name="text">
           <string>首页</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="ptn_last">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_now">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ptn_next">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="ptn_end">
          <property name="text">
           <string>尾页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ptn_NotCorrected">
          <property name="text">
           <string>未批改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ptn_released">
          <property name="text">
           <string>已销假</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ptn_writeOff">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>330</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>销假</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>931</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: #BDC8E2;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}

</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_isSchool">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>是否离校</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_leaveType">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>请假类型</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>471</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_timeBegin">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>开始时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_timeEnd">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>结束时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>350</width>
        <height>130</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>130</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>348</width>
         <height>128</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_leaveReason_2">
          <property name="minimumSize">
           <size>
            <width>330</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>330</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>请假原因</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageSwitch</class>
   <extends>QWidget</extends>
   <header>imageswitch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
